Component.hpp

```diff
@@ -28,21 +33,27 @@ namespace spic {
             /**
              * @brief The parent of this component
              * @param parent the current parent
+             * @sharedapi
              */
-            std::weak_ptr<spic::GameObject> GameObject() const { return _gameObject; }
+            std::weak_ptr<spic::GameObject> GameObject() const;
 
             /**
              * @brief The parent of this component
              * @param parent the new parent
+             * @sharedapi
              */
-            void GameObject(std::weak_ptr<spic::GameObject> gameObject) { _gameObject = gameObject; }
+            void GameObject(std::weak_ptr<spic::GameObject> gameObject);
```

Comment:
parameters in de documentatie kloppen niet

Reply:
Geupdate